css/style.css

```diff
@@ -1 +1,1461 @@
-body {}
\ No newline at end of file
+:root {
+  --btn-primary-color: rgba(33, 231, 134, 1);
```

Comment:
If opacity is 1 no need to use `rgba()`, use `rgb()` instead

Reply:
#fixed